File: fastframe/src/main/java/com/code/fastframe/baseactivity/BaseBundleActivity.java
package com.code.fastframe.baseactivity;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.code.fastframe.ciface.IBaseBundle;

public abstract class BaseBundleActivity extends BaseEventActivity implements IBaseBundle {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getBunleData();
    }

    @Override
    public void getBunleData() {

    }
}

File: fastframe/src/main/java/com/code/fastframe/baseactivity/BaseBundleActivity.kt
package com.code.fastframe.baseactivity

import android.os.Bundle
import com.code.fastframe.ciface.IBaseBundle
import com.code.fastframe.eventbus.GlobalEvent

abstract class BaseBundleActivity : BaseEventActivity(), IBaseBundle {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    getBunleData()
  }

  override fun getBunleData() {}
  override fun eventComming(globalMsg: GlobalEvent) {
    super.eventComming(globalMsg)
  }
}